Dockerfile.cypress

```diff
@@ -0,0 +1,68 @@
+##################################################
+#
+# cypress image
+FROM cypress/included:12.17.4
```

Comment:
This will need to be updated once https://github.com/openshift/console/pull/13887 merges